app.js

```diff
@@ -0,0 +1,42 @@
+var express = require('express');
+var app = express();
+
+// mongoose is just another node module
+var mongoose = require('mongoose');
+var db = mongoose.connect('mongodb://localhost/trucksAPI');
+var bodyParser = require('body-parser');
+
+var Truck = require('./models/truckModel');
+var idManager = require('./idManager');
+
+idManager.setIds(trucks.getTrucks());
+
+app.use(express.router('Routes', truckRoutes));
```

Comment:
This isn't how you use the router module. It should mount on a path (first argument) and specify the module as the second argument.